### PR TITLE
Fixed the UI issue in the application

### PR DESCRIPTION
**Pull request by ashvin89**
This PR addresses issue #72 

|PR checklist|Status|
|-----|:------:|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA|
|Is code analysis task executed and issues resolved in the CPP/native code?|Yes|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA|
|Is code commenting done properly for the changes made?|Yes|
|Is correct variable naming convention followed?|Yes|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|Yes|
|Are the message strings used in the application taken from strings.xml file?|NA|
|Are the permissions which are not required removed from AndroidManifest.java file?|NA|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA|
|Are the generated files correctly configured in .gitignore?|NA|
|Are the release signing configurations removed from the gradle files?|NA|
|Is commented code removed from all files?|Yes|
|Are updated images compressed (< 100 KB)?|NA|
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|Yes|

### CODE REVIEW

File: EnglishWebRoot/js/viewIndex.js [hunk 1]
```diff
@@ -2,17 +2,11 @@ document.addEventListener("deviceready", function () {
     viewportFix();
 
     // for Google Pixel C tab show navigation bar
```
Comment (Shailin) [inline EnglishWebRoot/js/viewIndex.js]: Update the comment

File: EnglishWebRoot/js/viewSchool.js [hunk 2]
```diff
@@ -131,9 +131,18 @@ document.addEventListener("deviceready", function () {
...
                 $(elem).attr("subject", cur.name);
-				$(elem).attr("id", cur.name);
+                $(elem).attr("id", cur.name);
                 if (cur.icon) {
                     $(elem).find("img").attr("src", window.schoolLoc + "/" + cur.name + "/icons/" + cur.icon);
+
+                    var heightdiff = document.body.clientHeight - window.outerHeight;
+                    var ratio = (heightdiff / document.body.clientHeight) * 100;
+                    var originalposition = 50 - ratio;
+                    var img9 = $(elem).find("img");
```
Comment (Shailin) [inline EnglishWebRoot/js/viewSchool.js]: Update variable name

File: EnglishWebRoot/school/script/mouse.js [hunk 3]
```diff
@@ -247,6 +270,8 @@ function addUserControl() {
                     mMode = "line";
                 } else if (workspaceRet) {
                     if (!curSequence || curInterruptTime == 0) {
+                        // Try to fix drawline
```
Comment (Shailin) [inline EnglishWebRoot/school/script/mouse.js]: Remove unwanted comments

File: EnglishWebRoot/school/script/load.js [hunk 4]
```diff
@@ -3497,23 +3497,44 @@ function getValue(parent, nodeName, silentError) {
...
         document.head.firstChild.content = 'width=device-width, initial-scale=' + viewportScale + ', maximum-scale=' + viewportScale + ', minimum-scale=' + viewportScale + ', user-scalable=no';
-//	vertCenterBook(curScreenHeight);
-        /*
-         $("#background").css({"height":0,"width":0});
-         $("#background").css({"height":"100%","width":"100%"});
-         */
+        // Fitting images to all the device according to their screen width and screen height 
+        window.setTimeout(function () {
+            if ((screenHeight / maxDim[0]) < (screenWidth / maxDim[1])) {
+                screenAttributes = document.getElementById('bookC');
```
Comment (Shailin) [inline EnglishWebRoot/school/script/load.js]: Move screenAttributes outside if statement

File: EnglishWebRoot/js/viewIndex.js (hunk 1 above)
Comment (ashvin89) [reply EnglishWebRoot/js/viewIndex.js]: Updated the comment

File: EnglishWebRoot/js/viewSchool.js (hunk 2 above)
Comment (ashvin89) [reply EnglishWebRoot/js/viewSchool.js]: Updated variable name

File: EnglishWebRoot/school/script/mouse.js (hunk 3 above)
Comment (ashvin89) [reply EnglishWebRoot/school/script/mouse.js]: Removed unwanted comments

File: EnglishWebRoot/school/script/load.js (hunk 4 above)
Comment (ashvin89) [reply EnglishWebRoot/school/script/load.js]: Moved screenAttributes outside

File: EnglishWebRoot/js/viewSchool.js [hunk 5]
```diff
@@ -208,13 +218,13 @@ document.addEventListener("deviceready", function () {
...
             $.ajax("../ajax/set/setSchoolRowAndColCount.php?school=" + window.schoolLoc + "&rows=" + subjectRowNum + "&cols=" + subjectColNum).done(
-              function (ret) {
-                  if (ret == "done") {
-                      webix.message("Rows modified");
-                  } else {
-                      // document.body.innerHTML = ret;
-                  }
-              });
+                function (ret) {
+                    if (ret == "done") {
```
Comment (ankitk20) [inline EnglishWebRoot/js/viewSchool.js]: Use ===

File: EnglishWebRoot/school/script/mouse.js [hunk 6]
```diff
@@ -54,26 +72,26 @@ function animPlate(mod) {
                     "-o-transform": rightVal
                 });
                 if (dir == 'down') {
```
Comment (ankitk20) [inline EnglishWebRoot/school/script/mouse.js]: Use ===

File: EnglishWebRoot/school/script/mouse.js [hunk 7]
```diff
@@ -205,6 +223,10 @@ function addUserControl() {
             } else {
                 pos = [event[mxStr] + scrollOffsets[1], event[myStr] + scrollOffsets[0] - vertCenteredOffset];
             }
+            if ((screenHeight / maxDim[0]) < (screen.width / maxDim[1]))
+                pos[0] = settingPointOnScreen(pos[0], screen.width, maxDim[1], document.body.clientWidth / screen.width)
+            else
+                pos[1] = settingPointOnScreen(pos[1], screen.height, maxDim[0], document.body.clientHeight / screen.height)
```
Comment (ankitk20) [inline EnglishWebRoot/school/script/mouse.js:230]: Use braces

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: EnglishWebRoot/js/viewSchool.js (hunk 5 above)
Comment (ashvin89) [reply EnglishWebRoot/js/viewSchool.js]: Used ===

File: EnglishWebRoot/school/script/mouse.js (hunk 7 above)
Comment (ashvin89) [reply EnglishWebRoot/school/script/mouse.js:230]: Used braces

File: EnglishWebRoot/school/script/mouse.js (hunk 6 above)
Comment (ashvin89) [reply EnglishWebRoot/school/script/mouse.js]: Used ===

**Review by ankitk20 — DISMISSED** [review]

**Review by Shailin — DISMISSED** [review]

File: EnglishWebRoot/school/pubbly_engine/latest/js/run/dom.js [hunk 8]
```diff
@@ -152,6 +155,67 @@ function PubblyDom(xml, environment) {
...
+            var totalpixel = parseFloat(window.getComputedStyle(screenAttributes).width);
+            screenAttributes.style.width = domWidth;
+            //  Calculating Scale % to increase the width of the screen
+            return (totalpixel / info.width);
+        }
+
+        window.setTimeout(function (screenAttributes) {
+            var scalingFactor = 0, translatingFactor = 0;
+
+            screenAttributes = document.getElementsByClassName('transformCenter')[0];
```
Comment (mathewnik90) [inline EnglishWebRoot/school/pubbly_engine/latest/js/run/dom.js:179]: This method can be written better.
Why do you need the set Timeout?

File: EnglishWebRoot/school/pubbly_engine/latest/js/run/_Pubbly.js [hunk 9]
```diff
@@ -78,6 +79,28 @@ class Pubbly {
 			what = [what];
 		}
 		let found = [];
+		// Calculating the viewport scale
```
Comment (mathewnik90) [inline EnglishWebRoot/school/pubbly_engine/latest/js/run/_Pubbly.js:82]: Is it possible to move the file to a common location?

**Review by mathewnik90 — DISMISSED** [review]

File: EnglishWebRoot/school/pubbly_engine/latest/js/run/dom.js (hunk 8 above)
Comment (ashvin89) [reply EnglishWebRoot/school/pubbly_engine/latest/js/run/dom.js:179]: Refactored the code.
setTimeout() function is required to correctly fetch the viewport properties. Tested after removing the property.

File: EnglishWebRoot/school/pubbly_engine/latest/js/run/_Pubbly.js (hunk 9 above)
Comment (ashvin89) [reply EnglishWebRoot/school/pubbly_engine/latest/js/run/_Pubbly.js:82]: Not considering it as discussed.

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**